Add documentation to register an event subscriber for mongodb odm bundle (#2393)
doctrine-extensions/DoctrineExtensions: doc/symfony4.md

behaviors. This listener will set the **locale used** from request and **username** to
loggable and blameable. So, to finish the setup create **EventSubscriber\DoctrineExtensionSubscriber**

> ## Register event subscriber for [Symfony Doctrine MongoDB Bundle](https://packagist.org/packages/doctrine/mongodb-odm-bundle)

Reviewer: ```suggestion
## Register event subscriber for [Symfony Doctrine MongoDB Bundle](https://github.com/doctrine/DoctrineMongoDBBundle)
```

Author: Changed!

> Because DoctrineExtensions does not implement EventSubscriberInterface from MongoDBBundle, you need to manually tag 

Reviewer 2: ```suggestion
Because DoctrineExtensions does not implement `EventSubscriberInterface` from MongoDBBundle, you need to manually tag 
```

> the listeners. Otherwise, they will not be triggered by Doctrine.

Reviewer 2: I think we should improve this last sentence regarding the implicit subject, as the listeners are not triggered (the listeners are listening by the triggered events).

> the listener. Otherwise, the LoggableListener will not be triggered by Doctrine.

Reviewer: ```suggestion
the listeners. Otherwise, they will not be triggered by Doctrine.
```
The idea is that this affects not only `LoggableListener` but all listeners

> ```php

Reviewer 2: ```suggestion
```yaml
```
